mne/evoked.py

```diff
@@ -406,13 +406,17 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             Color code lines by mapping physical sensor coordinates into color
             values. Spatially similar channels will have similar colors.
             Bad channels will be dotted.
+        legend : bool
+            Whether to show legend when using ``spatial_colors``. If
+            ``spatial_colors=False``, this parameter has no effect. Defaults
+            to True.
         """
         return plot_evoked(self, picks=picks, exclude=exclude, unit=unit,
                            show=show, ylim=ylim, proj=proj, xlim=xlim,
                            hline=hline, units=units, scalings=scalings,
                            titles=titles, axes=axes, gfp=gfp,
                            window_title=window_title,
-                           spatial_colors=spatial_colors)
+                           spatial_colors=spatial_colors, legend=legend)
```

Comment:
>
> spatial_colors : bool | str
>
>     if str, must be one of 'head', 'skirt', 'notopo'
>
>
ok that can work. Then I would do

spatial_colors = None | 'head' | 'skirt' | 'notopo'

so spatial_colors becomes str or None